conda-forge.yml

```
@@ -6,3 +6,9 @@ os_version:
   linux_64: cos7
 conda_build:
   pkg_format: '2'
+build_platform:
+  osx_arm64: osx_64
+  linux_aarch64: linux_64
```

```suggestion
  linux_aarch64: linux_aarch64
```

And you can switch between azure or Travis below